dts/arm/st/u0/stm32u0.dtsi

```diff
@@ -207,6 +208,42 @@
 			interrupts = <29 0>;
 			status = "disabled";
 		};
+
+		i2c1: i2c@40005400 {
+			compatible = "st,stm32u0-i2c", "st,stm32-i2c-v2";
+			clock-frequency = <I2C_BITRATE_STANDARD>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40005400 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00200000>;
+			interrupts = <23 0>;
+			interrupt-names = "combined";
+			status = "disabled";
+		};
+
+		i2c2: i2c@40005800 {
+			compatible = "st,stm32u0-i2c", "st,stm32-i2c-v2";
+			clock-frequency = <I2C_BITRATE_STANDARD>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40005800 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00400000>;
+			interrupts = <24 0>;
+			interrupt-names = "combined";
+			status = "disabled";
+		};
+
+		i2c3: i2c@40008800 {
+			compatible = "st,stm32u0-i2c", "st,stm32-i2c-v2";
+			clock-frequency = <I2C_BITRATE_STANDARD>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40008800 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00800000>;
+			interrupts = <24 0>;
```

Comment:
Question here.
It seems that instances 2, 3 and 4 will share the same interrupt line. Issue is that there is nothing to handle this case from what I can see at first sight.
Please confirm and see how to address:
- add support using SHARED_IRQ
- generate an error on unsupported configuration
- ...